features/270-next-skycli/commands.md

```diff
@@ -246,6 +232,85 @@ $ skycli app view-tenant-config > ./tenant-config.yaml
 $ skycli app update-tenant-config -f ./tenant-config.yaml
 ```
 
+### skycli app deploy
+
+`skycli app deploy [NAME]`
+
+Deploy cloud functions by reading skygear.yaml config file. `[NAME]` is the name
+of deployment items in the config file. If `[NAME]` is missing, the commands will
+update everything based on the config.
+
+#### Example
+
+Given that we have `skygear.yaml` like this,
+
+```
+app: myapp
+deployments:
+  function1:
+    type: http-handler
+    path: /hello-world
+    env: node
+    src: js/hellow-world
+    secrets:
+      - DATABASE_URL
+    permission:
+      - name: key_required
+      - name: user_required
+  static-index:
+    type: static
+    src: index.html
+    path: /
+  static-asset:
+    type: static
+    src: asset
+    path: /static
+```
+
+1. Running `skycli app deploy`, all deployment items will be deployed.
```

Comment:
The synopsis

```sh
skycli app deploy < --all | ITEM_NAME >
```

is safer since user may expect `skycli app deploy` to print help.

Reply:
Sure, updated!